pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -66,7 +68,8 @@ func NewMustGatherCommand(f kcmdutil.Factory, streams genericclioptions.IOStream
 	}
 
 	cmd.Flags().StringVar(&o.NodeName, "node-name", o.NodeName, "Set a specific node to use - by default a random master will be used")
-	cmd.Flags().StringVar(&o.Image, "image", o.Image, "Set a specific image to use, by default the OpenShift's must-gather image will be used.")
+	cmd.Flags().StringSliceVar(&o.Images, "image", o.Images, "Specify a must-gather plugin image to run. If not specified, OpenShift's default must-gather image will be used.")
+	cmd.Flags().StringSliceVar(&o.ImageStreams, "image-stream", o.ImageStreams, "Specify an image stream containing a must-gather plugin image to run.")
```

Comment:
This is a good catch for our rhhi colleagues.  I understand the purpose to provide stable command line across versions, but please add to the command line help so others will get it too. 

@sferich888 @sttts if the external teams engage you, I think this is the recommendation for them to have a stable CLI must-gather command line flag.

Reply:
@deads2k care to explain this a bit more (are you trying to say use --image-stream A,B,C)